build.cmd

```diff
@@ -0,0 +1,38 @@
+@echo off
+SETLOCAL
+
+
+.paket\paket.bootstrapper.exe
+if errorlevel 1 (
+  exit /b %errorlevel%
+)
+
+.paket\paket.exe restore
+if errorlevel 1 (
+  exit /b %errorlevel%
+)
+
+:Build
+cls
+
+SET FAKE_PATH=packages\build\FAKE\tools\Fake.exe
+SET Platform=
+
+IF [%1]==[] (
+    "%FAKE_PATH%" "build.fsx" "Default" 
+) ELSE (
+    "%FAKE_PATH%" "build.fsx" %* 
+) 
+
+if defined TEAMCITY_PROJECT_NAME goto Quit
```

Comment:
Depends on the build server this file will build on. Or if we keep the loop at all.